components/facets/custom-facets/audit-title.js

```diff
@@ -0,0 +1,34 @@
+// node_modules
+import PropTypes from "prop-types";
+// components
+import { auditMap } from "../../audit";
+
+/**
+ * Displays the title for audit facets. These show a shortened title and an icon matching the icons
+ * in the audit panels. This component gets used for the titles of all audit categories.
+ */
+export default function AuditTitle({ facet }) {
+  // Extract the audit type from the facet field name, and use it to get the color and title.
+  const auditType = facet.field.split(".")[1];
```

Comment:
will audit names always have a `.`?

Reply:
So far. It was in all of encoded time. I could see the argument for not trusting that though.

Reply:
if the format is the same then it's all good for now